Sources/Apollo/GraphQLUpload.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+public protocol GraphQLUpload: JSONEncodable {
```

Comment:
Maybe rename this to `GraphQLFile`? Since conceptually, one "upload" can contain multiple files?

Reply:
I don't really understand the Swift implementation, but regarding naming I recommend maintaining consistency with other file upload implementations. The scalar is called `GraphQLUpload` because technically it is not a file, but a promise of a file being uploaded that might, or might not, finish uploading.